src/accelerate/accelerator.py

```diff
@@ -272,7 +272,7 @@ def __init__(
             self.native_amp = is_torch_version(">=", "1.6")
             if not self.native_amp:
                 raise ValueError(err.format(mode="fp16", requirement="PyTorch >= 1.6"))
-            if not torch.cuda.is_available():
+            if not torch.cuda.is_available() and os.environ.get("USE_MPS_DEVICE", "false") == "false":
```

Comment:
There might be other truthy values we're not catching here, can se use something like `parse_flag_from_env`?

Reply:
Done.